### PR TITLE
Refactor/customer service bot

### PR DESCRIPTION
**Pull request by ruthenian8**

### CODE REVIEW

**Review by avsakharov — APPROVED** [review]

File: customer_service_bot/bot/run.py [hunk 1]
```diff
@@ -0,0 +1,35 @@
+import os
+
+from dff.messengers.telegram import PollingTelegramInterface
+from dff.pipeline import Pipeline
+
+from dialog_graph import graph
```
Comment (RLKRo) [inline customer_service_bot/bot/run.py:6]: No such file.

File: customer_service_bot/bot/test.py [hunk 2]
```diff
@@ -3,8 +3,8 @@
 from dff.messengers.telegram import TelegramMessage
 from dff.script import RESPONSE, Message
 
-from dff_chatgpt_skill.main import script
-from dff_chatgpt_skill.pipeline import pipeline
+from dialog_graph.graph import script
```
Comment (RLKRo) [inline customer_service_bot/bot/test.py:6]: No such file.

File: customer_service_bot/README.md [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+echo OPENAI_API_TOKEN=*** >> .env
+```
+
+Build the bot:
+```commandline
+docker-compose build
+```
+Testing the bot:
+```commandline
+docker-compose run bot pytest test.py
```
Comment (RLKRo) [inline customer_service_bot/README.md:45]: No service named `bot`.

File: customer_service_bot/README.md [hunk 4]
```diff
@@ -0,0 +1,56 @@
...
+```
+
+Running the bot:
+```commandline
+docker-compose run bot python run.py
```
Comment (RLKRo) [inline customer_service_bot/README.md:50]: No service named `bot`.

**Review by RLKRo — CHANGES_REQUESTED** [review]

File: customer_service_bot/bot/api/chatgpt.py [hunk 5]
```diff
@@ -0,0 +1,81 @@
...
+
+CHATGPT_COHERENCE_PROMPT = """
+What follows is a question and an answer. Just write 'true' if the answer was satisfactory or 'false' otherwise.
+"""
+
+openai.api_key = os.getenv("OPENAI_API_TOKEN")
+
+
+def get_output_factory():
+    """Construct a get_output function encapsulating the execution counter.
```
[the remark this answers is not in the document]
Comment (ruthenian8) [reply customer_service_bot/bot/api/chatgpt.py]: Thx

File: customer_service_bot/bot/run.py [hunk 6]
```diff
@@ -0,0 +1,35 @@
...
+    openai_api_token = os.getenv("OPENAI_API_TOKEN")
+
+    if not openai_api_token:
+        raise RuntimeError("Openai api token (`OPENAI_API_TOKEN`) system variable is required.")
+
+    if telegram_token:
+        messenger_interface = PollingTelegramInterface(token=telegram_token)
+
+    elif interface_cli is not None:
+        messenger_interface = None
```
[the remark this answers is not in the document]
Comment (ruthenian8) [reply customer_service_bot/bot/run.py]: Thx

File: customer_service_bot/bot/test.py (hunk 2 above)
Comment (ruthenian8) [reply customer_service_bot/bot/test.py]: Thx

File: customer_service_bot/README.md (hunk 3 above)
Comment (ruthenian8) [reply customer_service_bot/README.md]: Thx

File: customer_service_bot/README.md (hunk 4 above)
Comment (ruthenian8) [reply customer_service_bot/README.md]: Thx

**Comment by RLKRo** [comment]
Oh, [the interface_cli issue](https://github.com/deeppavlov/dialog_flow_demo/pull/7#discussion_r1170542838) was introduced in #3 which I didn't notice during its review.
Could you also fix it in [faq_bot](https://github.com/deeppavlov/dialog_flow_demo/blob/main/frequently_asked_question_bot/bot/run.py)?

**Comment by ruthenian8** [comment]
> Oh, [the interface_cli issue](https://github.com/deeppavlov/dialog_flow_demo/pull/7#discussion_r1170542838) and the [no such file issue](https://github.com/deeppavlov/dialog_flow_demo/pull/7#discussion_r1170605950) were introduced in #3. Could you also fix them in faq_bot?

Done

**Review by RLKRo — APPROVED** [review]
I fixed some errors in FAQ bot, it now works. I fixed the same errors in customer service bot, but I don't have a token to test it. Could you make sure that customer service bot also works (run tests & run bot)?

**Review by kudep — APPROVED** [review]

**Pull request merged**